gui/src/hv/ram.rs

```diff
@@ -302,6 +302,12 @@ impl<'a> LockedAddr<'a> {
     }
 }
 
+impl AsRef<[u8]> for LockedAddr<'_> {
+    fn as_ref(&self) -> &[u8] {
+        unsafe { std::slice::from_raw_parts(self.ptr, self.len.get()) }
+    }
+}
+
```

Comment:
We can't do this because the returned slice subject to the data race, which is UB for Rust.